.mailmap

```diff
@@ -1609,6 +1610,7 @@ aditisingh2362 <aditisingh2362@gmail.com>
 alejandro <amartinhernan@gmail.com>
 alijosephine <alijosephine@gmail.com>
 amsuhane <ayushsuhane99@iitkgp.ac.in> amsuhane <“ayushsuhane99@iitkgp.ac.in”>
+anandkaranubc <119553199+anandkaranubc@users.noreply.github.com>
```

Comment:
There should not be two separate entries in the .mailmap file unless they are for two separate authors:
https://github.com/sympy/sympy/blob/b7e7de3b02c8d1edf683e12c61a69f57eebd4ce9/.mailmap#L129-L148